app-modules/knowledge-base/src/Filament/Resources/KnowledgeBaseItemResource/Pages/ViewKnowledgeBaseItem.php

```diff
@@ -56,6 +58,19 @@ public function getTitle(): string | Htmlable
         return $this->record->title;
     }
 
+    public function mount(int | string $record): void
```

Comment:
I think we should maybe do this on the edit page as well. What do you think?

Reply:
That wasn't part of the ticket, and it feels like editors aren't utilising the article so it shouldn't increment the views IMO

Reply:
Got it, makes sense.